### PR TITLE
[MI-1865] Fixed camera and microphone issue

### PR DESCRIPTION
**Pull request by raghavaggarwal2308**
1. Changed the "HTTP" protocol to "HTTPS".
2. Added code to keep the camera and microphone off by default.

### CODE REVIEW

File: webapp/src/components/conference/conference.tsx [hunk 1]
```diff
@@ -99,6 +99,10 @@ export default class Conference extends React.PureComponent<Props, State> {
             jwt: this.props.jwt,
             noSSL,
             parentNode: document.querySelector('#jitsiMeet'),
+            configOverwrite: {
+                startWithVideoMuted: true,
+                startWithAudioMuted: true
+            },
```
Comment (manojmalik20) [inline webapp/src/components/conference/conference.tsx]: Are these necessary for fixing the issue or is the issue just fixable by `https` change?

**Review by manojmalik20 — APPROVED** [review]

File: webapp/src/components/conference/conference.tsx (hunk 1 above)
Comment (raghavaggarwal2308) [reply webapp/src/components/conference/conference.tsx]: No these are not necessary for the fix.....I added this to keep the camera and microphone off by default.

Comment (manojmalik20) [reply webapp/src/components/conference/conference.tsx]: I don't think there's a requirement for this. You can remove this.

Comment (raghavaggarwal2308) [reply webapp/src/components/conference/conference.tsx]: okay

**Pull request merged**